react/Icon/index.jsx

```diff
@@ -101,4 +101,3 @@ Icon.defaultProps = {
 }
 
 export default Icon
-export { default as Sprite } from './Sprite'
```

Comment:
Je crois qu'on peut mettre la description du commit sur une seul ligne sans que les hook de prepush râle. Ca ferait une discription plus propre sur la page de release de Ui. Et il y a un `.` en trop entre `directly` et `with`.

On peut peut-être rajouter une bafouille sur le icon="xxxx" et le fait d'utiliser les svgr.

Ca donnerait : 

 `<Sprite>` is no longer exported by `<Icon>`. However, you can import it directly with `import Sprite from 'cozy-ui/transpiled/react/Icon/Sprite'`. Be carful, it's no longer possible to set an icon with a string like this `<Icon icon="myIcon" />` without importing `<Sprite>`. If don't need the sprite, you can still use svgr icons:
```
import MyIcon from 'cozy-ui/transpiled/react/Icons/MyIcon'
<Icon icon={MyIcon} />
```